src/main/java/domain/CardDeck.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+public class CardDeck {
+
+    private static final String CARD_DECK_IS_EMPTY_EXCEPTION_MESSAGE = "카드덱에 카드가 남아있지 않습니다.";
+    private List<Card> cardDeck;
+
+    public CardDeck() {
+        cardDeck = new LinkedList<>();
+        for (Type type : Type.values()) {
+            for (Symbol symbol : Symbol.values()) {
+                cardDeck.add(new Card(type, symbol));
+            }
+        }
+        Collections.shuffle(cardDeck);
+    }
+
+    public Card draw() {
+        if (cardDeck.size() == 0) {
```

Comment:
```suggestion
        if (cardDeck.isEmpty()) {
```

해당 기능을 하는 API를 제공하고 있습니다!

Reply:
수정하였습니다:)